### DISCUSSION TITLE
Vanilla RN: Unrecognized font family 'Inter' when running via Xcode

### DISCUSSION THREAD

**Discussion by wan54** [tamagui/tamagui#716] [General]
Hi,

Is there a doco that explain how to install Inter font in Xcode?

tamagui.config.ts

```
import {config} from '@tamagui/config';

import {createTamagui} from '@tamagui/core';
import {TamaguiInternalConfig} from 'tamagui';

const appConfig: TamaguiInternalConfig = createTamagui(config);

export type AppConfig = typeof appConfig;

declare module '@tamagui/core' {
  // overrides TamaguiCustomConfig so your custom types
  // work everywhere you import `tamagui`
  interface TamaguiCustomConfig extends AppConfig {}
}

export default appConfig;
```

**Comment by LunatiqueCoder** [comment]
1. You'll need to create a `react-native.config.js` in the root folder like this:

```javascript
module.exports = {
  project: {
    ios: {},
    android: {},
  },
  assets: [
    './node_modules/@tamagui/font-inter/otf',
    './node_modules/@tamagui/font-silkscreen/files',
  ],
};
```
Example code: https://github.com/criszz77/luna/blob/master/template/react-native.config.js



-------

2. Then run `yarn react-native-asset` in your terminal.



-------
3. Then use them in your Tamagui config like this: `Inter-Medium`, `Inter-Bold`, etc...

Example code: https://github.com/criszz77/luna/blob/master/template/tamagui/fonts.ts#L87-L89

----

#### **EDIT**: 
This also has been answered here: https://github.com/tamagui/tamagui/discussions/487
